test/unit/LedgerId.js

```diff
@@ -0,0 +1,117 @@
+import * as hex from "../src/encoding/hex.js";
+import LedgerId from "../src/LedgerId.js";
+
+const MAINNET = "mainnet";
+const HEX_MAINNET = hex.encode(new Uint8Array([0]));
+
+const TESTNET = "testnet";
+const HEX_TESTNET = hex.encode(new Uint8Array([1]));
+
+const PREVIEWNET = "previewnet";
+const HEX_PREVIEWNET = hex.encode(new Uint8Array([2]));
+
+const OTHER = "other";
+const HEX_OTHER = hex.encode(new Uint8Array([3]));
+
+describe("LedgerId", function () {
+    it('fromString returns LedgerId from strings/hex "mainnet", "testnet", "previewnet"', function () {
+        expect(LedgerId.fromString(MAINNET).toString()).to.eql(MAINNET);
```

Comment:
Not blocking the PR for this, but I might be better to save `LedgerId.fromString(MAINNET)` into a variable and compare it. I'm gong to assume that calling `LedgerId.fromString()` is slower. That shouldn't matter too much here as I think it'll still be fast, but just for future.